### PR TITLE
Rename cell type, other dataset config files fixes

### PR DESCRIPTION
**Pull request by daniel-ji**
cell_type.name umbilical vein endothelial cell -> endothelial cell of umbilical vein to match what's stored online (and pass validation, since there's no fuzzy searching planned) see https://www.ebi.ac.uk/ols4/ontologies/cl/classes/http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0002618

Other various changes that make config files pass validation. 

Also updated validation to allow for lowercase letters in the ontology prefix.

### CODE REVIEW

**Review by jgadling — APPROVED** [review]

File: ingestion_tools/dataset_configs/10008.yaml [hunk 1]
```diff
@@ -192,7 +192,7 @@ tiltseries:
       energy_filter: GIF Quantum LS
       phase_plate: null
     pixel_spacing: 3.52
-    related_empiar_entry: EMD-15053
```
Comment (manasaV3) [inline ingestion_tools/dataset_configs/10008.yaml:195]: @uermel is there a reason we want this to capture just the EMPIAR values for tiltseries and not include the EMD entries?

File: ingestion_tools/dataset_configs/10001.yaml [hunk 2]
```diff
@@ -372,7 +372,7 @@ datasets:
       corresponding_author_status: true
       name: Judith B. Zaugg
     cell_strain:
-      id: https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=284812
+      id: NCBITaxon:284812
```
Comment (manasaV3) [inline ingestion_tools/dataset_configs/10001.yaml:375]: We should co-ordinate with the FE engineers about this change, as they use this id for constructing the link. 
For example: https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=284812 has the relevant value we are looking for. But https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=NCBITaxon:284812  doesn't.

**Review by manasaV3 — APPROVED** [review]
LGTM, please link the issue to track the FE fix for the ontology update. :)

File: ingestion_tools/dataset_configs/10001.yaml (hunk 2 above)
Comment (daniel-ji) [reply ingestion_tools/dataset_configs/10001.yaml:375]: Sounds good!

**Pull request merged**